docs/content/release_notes/changelog_1.10.0.md

```diff
@@ -1,4 +1,5 @@
 # 1.10.0 (July 2022)
 
 IBM® block storage CSI driver 1.10.0 adds new support and enhancements:
+- Support for Spectrum Virtualize Snapshot function that was introduced in IBM Spectrum Virtualize 8.5.1 release
```

Comment:
isn't it supposed to be 8.5.1.0?

Reply:
no. 8.5.1 - per SV documentation as well
![image](https://user-images.githubusercontent.com/54570654/176657912-5e201966-53d7-44a9-a076-ed0e62430bad.png)